components/stories-tree/stories-tree.js

```diff
@@ -24,28 +25,52 @@ class StoriesTree extends HTMLElement {
       fireEvent('story-changed', this, { story, storyKind });
     });
 
+    delegate.on('keyup', this, '[ref="filter-input"]', (event, { value }) => {
+      const targetValue = value.toLowerCase();
+
+      this.$refs.storyList.forEach((story) => {
+        if (!~story.getAttribute('story').toLowerCase().indexOf(targetValue)) {
+          story.classList.add(HIDE_CLASS);
+        } else {
+          story.classList.remove(HIDE_CLASS);
+        }
+      });
+    });
   }
 
-  render(stories){
-    this.innerHTML = storiesTreeTemplate({ stories });
+  render(stories) {
+    stories = this.sortStoriesKeys(stories);
+    this.innerHTML = storiesTreeTemplate({ stories, placeholder: PLACEHOLDER });
+
+    if (!this.$refs) {
```

Comment:
this condition is redundant. doesn't brake anything, but doesn't do anything either. it's always true

Reply:
right